src/main/java/generator/Model.java

```diff
@@ -43,6 +43,136 @@
     private RandomPositioner randomPositioner;
     private SizeManager sizeManager = new SizeManager();
 
+    public static final class Builder{
```

Comment:
Jeden problem jaki mam z builderem jest taki, że jeżeli ktoś chce go użyć, to może nie zbudować do końca modelu, może pominąć np drivera.

Reply:
Mogę dodać w build sprawdzenie czy driver nie jest nullem, tak sprawdzam scenario
![image](https://user-images.githubusercontent.com/47008811/97912015-7b670080-1d4c-11eb-8d29-38ac5a62093c.png)

Reply:
tak, ale musiałbyś sprawdzić to dla wszystkich i wtedy cały sens buildera przepada, driver podałem jako przykład